lfs/core/management/commands/lfs_migrate.py

```diff
@@ -285,13 +293,19 @@ def migrate_to_09(self, application, version):
         old_criteria = ", ".join([str(row[0]) for row in cursor1.fetchall()])
 
         content_type = ContentType.objects.get_for_model(CartPriceCriterion)
+
+        # alter price column to avoid *** IntegrityError: null value in column "price" violates not-null constraint
+        cursor2.execute("""ALTER TABLE criteria_cartpricecriterion ALTER COLUMN price DROP NOT NULL""")
```

Comment:
Have you tried if adding null=True to this line: https://github.com/diefenbach/django-lfs/blob/master/lfs/core/management/commands/lfs_migrate.py#L282 works? I'd prefer such solution over plain SQL.

```db.alter_column('criteria_cartpricecriterion', 'price', models.FloatField(default=0.0, null=True))```